### PR TITLE
MEPTS-900 | List of Patients Defaulters or IIT - Template v1.3

### PR DESCRIPTION
**Pull request by rupansana12**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsDefaultersOrIITReport.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public String getVersion() {
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Template_List_Patients_Defaulters_IIT_TARV_v1.2.xls",
+              "Template_List_Patients_Defaulters_IIT_TARV_v1.3.xls",
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsDefaultersOrIITReport.java]: Can we change the name of the template to **List_Patients_Defaulters_IIT_TARV_v1.3.xls**?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsDefaultersOrIITReport.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public String getVersion() {
...
               "Template List Patients Defaulters IIT TARV Report",
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsDefaultersOrIITReport.java]: Same as the previous comment. We remove the word **Template** from the name

**Review by ItsMurumba — CHANGES_REQUESTED** [review]
Hey, @rupansana12 I left some comments on the PR. Kindly address them. Thanks

**Review by ItsMurumba — COMMENTED** [review]
Looks okay. Ready to be merged.

**Review by ItsMurumba — APPROVED** [review]

**Review by cfaife — APPROVED** [review]

**Pull request merged**